subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceMetrics.kt

```diff
@@ -0,0 +1,136 @@
+
+    override fun onGetSdkPropertySuccess(attempt: Int, api: Int) {
+        statsDSender.send(prefix, CountMetric("get-sdk-property.success"))
+    }
+
+    override fun onGetSdkPropertyAttemptFail(attempt: Int) {
+        statsDSender.send(prefix, CountMetric("get-sdk-property.attempt-fail"))
+    }
+
+    override fun onGetSdkPropertyFailure(throwable: Throwable) {
+        statsDSender.send(prefix, CountMetric("get-sdk-property.failure"))
+    }
+
+    override fun onInstallApplicationSuccess(device: Device, attempt: Int, applicationPackage: String) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("install-application.success"))
+    }
+
+    override fun onInstallApplicationAttemptFail(device: Device, attempt: Int, applicationPackage: String) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("install-application.attempt-fail"))
+    }
+
+    override fun onInstallApplicationFailure(device: Device, applicationPackage: String, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("install-application.failure"))
+    }
+
+    override fun onGetAliveDeviceSuccess(device: Device, attempt: Int) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("get-alive-device.success"))
+    }
+
+    override fun onGetAliveDeviceAttemptFail(device: Device, attempt: Int) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("get-alive-device.attempt-fail"))
+    }
+
+    override fun onGetAliveDeviceFailed(device: Device, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("get-alive-device.failure"))
+    }
+
+    override fun onClearPackageSuccess(device: Device, attempt: Int, name: String) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("clear-package.success"))
+    }
+
+    override fun onClearPackageAttemptFail(device: Device, attempt: Int, name: String, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("clear-package.attempt-fail"))
+    }
+
+    override fun onClearPackageFailure(device: Device, name: String, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("clear-package.failure"))
+    }
+
+    override fun onPullSuccess(device: Device, from: Path, to: Path) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("pull.success"))
+    }
+
+    override fun onPullAttemptFail(device: Device, attempt: Int, from: Path, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("pull.attempt-fail"))
+    }
+
+    override fun onPullFailure(device: Device, from: Path, throwable: Throwable) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("pull.failure"))
+    }
+
+    override fun onClearDirectorySuccess(device: Device, remotePath: Path, output: String) {
+        statsDSender.send(prefixWithDevice(device), CountMetric("clear-directory.success"))
```

Comment:
I see an analogy with HTTP endpoint. We want to know the status and time.
It'd be helpful to know both of them.

Can we send Timer instead of Count? 
If I'm not mistaken statsd aggregate `.count` will give us the count of events but we will know different percentiles of time. It will help to choose timeouts based on data.